src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -121,12 +123,19 @@ public function enable_widget_copy_paste( $result, $server, $request ) {
 		$route = $request->get_route();
 		
 		// Check if this matches our target endpoint.
-		if ( ! preg_match( '#^/wp/v2/widget-types/([a-zA-Z0-9_-]+)/encode$#', $route, $matches ) ) {
+		if (
+			! preg_match( '#^/wp/v2/widget-types/([a-zA-Z0-9_-]+)/encode$#', $route, $matches_encode )
+			&& ! preg_match( '#^/wp/v2/widget-types/([a-zA-Z0-9_-]+)/render$#', $route, $matches_render )
+		) {
```

Comment:
This could be a single regex, and then you don't need the different version of `$matches` below.

```suggestion
		if ( ! preg_match( '#^/wp/v2/widget-types/([a-zA-Z0-9_-]+)/(?:encode|render)$#', $route, $matches ) ) {
```

Sample pattern proof: https://regex101.com/r/y3nan7/1